.github/actions/grafana/action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Grafana'
+description: 'Grafana images'
+inputs:
+  tags:
+    description: 'Tags'
+    required: false
+    default: ''
+  labels:
+    description: 'Labels'
+    required: true
+  image:
+    description: 'Image target'
+    required: true
+    default: 'myteslamate/grafana'
+runs:
+  using: "composite"
+  steps:
+    - name: Docker meta
+      id: docker_meta
+      uses: docker/metadata-action@v5
+      with:
+        images: ${{ inputs.image }}
+        tags: ${{ inputs.tags }}
+        labels: ${{ inputs.labels }}
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+    - name: Build and push
```

Comment:
Can we really do without it?

```
 - name: Login to DockerHub
        uses: docker/login-action@v3.0.0
        with:
          username: teslamate
          password: ${{ secrets.DOCKER_PASSWORD }}
```

Reply:
It seems to work (I do it in the workflow before this action)